Android/app/src/main/java/io/github/project_travel_mate/login/LoginActivity.java

```diff
@@ -212,4 +222,43 @@ public void showMessage(String message) {
         Toast.makeText(this, message, Toast.LENGTH_LONG)
                 .show();
     }
+
+    @Override
+    public void validateSignUp() throws ValidationException {
+        if(name.getText().toString().equals("")){
+            throw new ValidationException("first name is required!");
```

Comment:
f Capital. and define these strings in `strings.xml`
We don't want constant strings here.

Do it for other error messages too.